core/src/main/scala/org/ensime/indexer/domain.scala

```diff
@@ -108,7 +108,7 @@ case class ArrayDescriptor(fqn: DescriptorType) extends DescriptorType {
   }
   def internalString = "[" + fqn.internalString
 }
-case class Descriptor(params: List[DescriptorType], ret: DescriptorType) {
+case class Descriptor(params: Seq[DescriptorType], ret: DescriptorType) {
```

Comment:
Whoops forgot about that one, was mainly because I was struggling with PB and got a immutable seq.  will have a look.